question_answering_based_on_context_using_openai_and_pinecone/README.md

```diff
@@ -0,0 +1,47 @@
+# Question Answering based on Context using OpenAI GPT-3 and Pinecone 
+
+This sample uses OpenAI GPT-3 and Pinecone to answer questions on a specific topic based on provided documents.
+
+## Use case
+Stores content/documentation relating to a specific topic obtained from a Google sheet into a Pinecone vector database and when a question is asked, identify related contexts to the question and generate an answer from OpenAI GPT-3 models. 
```

Comment:
Check the english language of this paragraph, seems not that clear

Reply:
Modified in https://github.com/ballerina-guides/ai-samples/pull/66/commits/d9c08c77460527472ba049ee200a979dba45dfef.